plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointColor.tsx

```diff
@@ -132,35 +104,17 @@ const PointColor: React.FC<BaseFieldProps<"pointColor">> = ({ value, editMode, o
               </FieldValue>
             </Field>
           )} */}
-          <Field gap={8}>
-            <FieldTitle>if</FieldTitle>
-            <FieldValue>
-              <Dropdown overlay={menu} placement="bottom" trigger={["click"]}>
-                <StyledDropdownButton>
-                  <p style={{ margin: 0 }}>{c.condition.operand}</p>
-                  <Icon icon="arrowDownSimple" size={12} />
-                </StyledDropdownButton>
-              </Dropdown>
-            </FieldValue>
-            <FieldValue>
-              <Dropdown overlay={menu} placement="bottom" trigger={["click"]}>
-                <StyledDropdownButton>
-                  <p style={{ margin: 0 }}>{c.condition.operator}</p>
-                  <Icon icon="arrowDownSimple" size={12} />
-                </StyledDropdownButton>
-              </Dropdown>
-            </FieldValue>
-            <FieldValue>
-              <NumberInput value={c.condition.value} />
-            </FieldValue>
-          </Field>
-          <Field>
-            <FieldTitle width={82}>色</FieldTitle>
-            <FieldValue>
-              <ColorBlock color={c.color} />
-              <TextInput value={c.color} />
-            </FieldValue>
-          </Field>
+          <ConditionField title="if" fieldGap={8} condition={c.condition} />
```

Comment:
It looks like this file still has commented out code that was being referenced from the Legend component. If you don't need it, please remove it.

Reply:
Done!